Lib/pdb.py

```diff
@@ -2204,10 +2208,15 @@ def main():
 
     opts = parser.parse_args()
 
-    if opts.m:
-        file = opts.m
+    if opts.module:
+        file = opts.module
         target = _ModuleTarget(file)
+        # If the module is given, the first positional argument is not the script
+        if opts.pyfile:
```

Comment:
```suggestion
        if opts.pyfile:
```

Maybe call this ``opt.arg0`` because we don't actually know that it is the ``pyfile``.

Reply:
Yeah that makes sense. I refactored this a bit, to only take an `args` for positional arguments. Added the help message in case the user is confused, and the code is a bit cleaner.